packages/frontend/src/pages/Contribute.jsx

```diff
@@ -70,18 +113,110 @@ export default observer(() => {
       const canvas = document.getElementById('cosmo')
       canvas.width = width
       canvas.height = height
+      const renderer = new THREE.WebGLRenderer({ antialias: true, canvas })
 
-      const ctx = canvas.getContext('2d')
-      ctx.font = '2rem Azeret Mono'
-      ctx.fillStyle = 'white'
-      ctx.textAlign = 'center'
-      ctx.fillText(
-        "Bhargov's cosmo should be here.",
-        canvas.width / 2,
-        canvas.height / 2
+      // set camera
+      let camera = new THREE.PerspectiveCamera(
+        // fov
+        35,
+        // aspect
+        width / height,
+        // near
+        0.01,
+        // far
+        10000 * AU
       )
+      const defaultZoom = 1.7 * AU
+      camera.position.set(0, 0, defaultZoom)
```

Comment:
Could we change the default angle of the view of the camera to let it look better? 
for example, observe from the side as the following image:
<img width="1312" alt="image" src="https://github.com/Unirep/trusted-setup/assets/9023842/53db7728-e63b-46d4-86b9-60c29be03bc3">